app/controllers/api/payment/braintree_controller.rb

```diff
@@ -8,8 +8,12 @@ def token
 
   def webhook
     webhook_notification = Braintree::WebhookNotification.parse(params[:bt_signature], params[:bt_payload])
```

Comment:
I'm all for refactoring existing code in this controller - but I'd rather leave that for another PR since this fix is quite urgent.